.github/workflows/checks.yml

```diff
@@ -83,12 +89,12 @@ jobs:
         with:
           persist-credentials: false
       - name: Initialize CodeQL
-        uses: github/codeql-action/init@461ef6c76dfe95d5c364de2f431ddbd31a417628 # v3.26.9
+        uses: github/codeql-action/init@df409f7d9260372bd5f19e5b04e83cb3c43714ae #  v3.27.9
         with:
-          config-file: ./.github/codeql.yml
-          languages: javascript
+          # config-file: ./.github/codeql.yml
```

Comment:
This job currently fails if this line is enabled because the configuration is not valid if `matrix.what == 'actions'`, but we want the configuration for `matrix.what == 'javascript'`...

Reply:
The reason for this is this line:

https://github.com/ericcornelissen/shescape/blob/7b572554c7d75a0e4710170c47fd59a0b8f20331/.github/codeql.yml#L6

which it complained about [here](https://github.com/ericcornelissen/shescape/actions/runs/12381546867/job/34560339987#step:5:631). 

This has been reported at https://github.com/github/codeql-action/issues/2658